### ISSUE TITLE
GDCPH Mini Heater Table Support

### ISSUE THREAD

**Issue by seanuleh** [Ralim/IronOS#1726]
I was looking to grab a MHP30 but across ali express, searching for MHP30 yields many results for this other device: https://www.aliexpress.com/item/1005005601132032.html

It's cheap and cheerful and if it supported IronOS it would be a no brainer. I found this german blog which has a teardown of the device; https://stepp-ke.de/2023/05/06/gdcph-mini-heating-table/

This blog indicates its using the following STC Microcontroller. Looking at the teardown pics, I cant see a serial to UART IC, and just trying various things I couldn't get it into a DFU boot mode; however, looks like the UART pins are exposed on the PCB so a potential pathway down that road. 

If technically possible, I think this would probably be a great little unit to port across as it ticks many boxes; USB-C PD powered, 5x5 surface area (larger than the MHP30) and best of all friggen cheap!

**Comment by Ralim** [comment]
Closing as duplicate of #1698 .
TL;DR basically never gonna get a proper port of IronOS to an 8051.
